experimental/telemetry_sink_test.go

```
@@ -0,0 +1,641 @@
+		WithResponseBodyMimeTypes([]string{"text/plain"})
+	cfg = experimental.WAFConfigWithTelemetrySink(cfg, sink)
+	cfg = experimental.WAFConfigWithPerRuleTiming(cfg, true)
+	waf, err := coraza.NewWAF(cfg)
+	if err != nil {
+		t.Fatalf("new waf: %v", err)
+	}
+	tx := waf.NewTransaction()
+	tx.ProcessURI("/x", "GET", "HTTP/1.1")
+	tx.AddRequestHeader("Host", "example.com")
+	tx.ProcessRequestHeaders()
+	tx.ProcessRequestBody()
```

golangci-lint (2.11.4)

[error] Error return value of `tx.ProcessRequestBody` is not checked

(errcheck)